src/patternfly/components/Select/select.scss

```diff
@@ -298,6 +298,7 @@
   color: var(--pf-c-select__menu-item--Color);
   text-align: left;
   white-space: nowrap;
+  background-color: transparent;
```

Comment:
I just submitted a PR that updates the shield. If that looks good, I think we can remove this property.

Reply:
I left it in and commented it. I tested with the updated shield, FF and FF nightly and I still see the gray background.